src/app/index.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
```

Comment:
Can we update this to be the icon we had before? (I think we had one)

Reply:
The icon before was generic placeholder, I did not add anything.